invenio_rdm_records/services/services.py

```diff
@@ -4,20 +4,39 @@
 # Copyright (C) 2020-2021 Northwestern University.
 # Copyright (C) 2021 TU Wien.
 # Copyright (C) 2021 Graz University of Technology.
+# Copyright (C) 2022 Universität Hamburg.
 #
 # Invenio-RDM-Records is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """RDM Record Service."""
 
+
+import tempfile
+
 import arrow
 from elasticsearch_dsl.query import Q
+from flask_iiif.api import IIIFImageAPIWrapper
 from invenio_drafts_resources.services.records import RecordService
-from invenio_records_resources.services import LinksTemplate
+from invenio_records_resources.services import LinksTemplate, Service
 from invenio_records_resources.services.uow import RecordCommitOp, unit_of_work
 
 from invenio_rdm_records.services.errors import EmbargoNotLiftedError
 
+try:
+    from importlib import metadata
+except ImportError:
+    # Running on pre-3.8 Python; use importlib-metadata package
+    import importlib_metadata as metadata
```

Comment:
Minor: You can just use ``import importlib_metadata as metadata`` as we're bound to it until at least 3.9 is EOL.